app/assets/javascripts/application.coffee

```diff
@@ -16,4 +16,3 @@ $(document).foundation()
 
 () ->
   $(document).foundation()
- 
```

Comment:
in application.coffee file we usually create ony function that called on document.ready event
you can define it simply like:

$ ->
  code here

Alse pls note some useful tricks from here
https://css-tricks.com/jquery-coffeescript/